packages/frontend/app/components/programs/root.js

```diff
@@ -41,8 +40,8 @@ export default class ProgramRootComponent extends Component {
   }
 
   get bestSelectedSchool() {
-    if (this.selectedSchoolId) {
-      return findById(this.args.schools.slice(), this.selectedSchoolId);
```

Comment:
This slice used to be needed as ember-data returned a thing that wasn't quite an array, however those days are gone and this can go away too and should do the same thing. Unless I'm wrong. Which isn't impossible.